### ISSUE TITLE
Bug: Python graphical watch doesn't work on Windows

### ISSUE THREAD

**Issue by JoostVisser** [awulkiew/graphical-debugging-vscode#14]
Hey, I tried to use the extension on Python but it didn't work for me.
I'm not sure if this is a Windows-only problem or that they changed the SessionType somewhere along the line.

OS: Windows
VSCode: v1.91.1

---

Steps to reproduce:
1. Install the latest version of VS Code.
2. Install Pylance
3. Debug a file (using Run and Debug)
4. Graphical Watch a variable.

Expected result: a plot with a line from (0,0) to (1,1)
Actual result: line unknown (LineString)

Sample code used.

```python
from shapely import LineString
line = LineString([(0, 0), (1, 1)])
print(line) # Breakpoint on this statement
```

---

**Solution:** It seems like the name of the vscode debugger is now `debugpy`.
Changing `debugger.ts:23` to include `debugpy` and everything works great.

**Comment by awulkiew** [comment]
It seems to work for me.

![obraz](https://github.com/user-attachments/assets/8ad87444-d674-46ad-a431-3e2a65aad5b9)
![obraz](https://github.com/user-attachments/assets/a3bd5d7d-f971-4830-9139-b118521ea873)

I have some problems with shapely multi-geometries but that's probably  a different story.
Which extension do you have installed for python debugging?

**Comment by awulkiew** [comment]
Regarding shapely multi-geometries, it seems that np.int32 is used to store the size and its value cannot be directly parsed as int.

![obraz](https://github.com/user-attachments/assets/361b79b3-0c0a-4dbe-a7b1-195bdb8a43d1)

**Comment by awulkiew** [comment]
I guess it's this one: https://marketplace.visualstudio.com/items?itemName=ms-python.debugpy
I'm testing with https://marketplace.visualstudio.com/items?itemName=ms-python.python
Thanks for the report and suggestion how to fix it.

**Comment by awulkiew** [comment]
Added/fixed in ver 0.9
